modules/flowable-dmn-engine/src/main/java/org/flowable/dmn/engine/impl/interceptor/DmnCommandInvoker.java

```diff
@@ -56,9 +56,7 @@ protected void executeOperations(CommandContext commandContext, boolean isStoreC
         DmnEngineAgenda agenda = CommandContextUtil.getAgenda(commandContext);
         while (!agenda.isEmpty()) {
             Runnable runnable = agenda.getNextOperation();
-            if (logger.isDebugEnabled()) {
-                logger.debug("Executing agenda operation {}", runnable);
-            }
+            LOGGER.debug("Executing agenda operation {}", runnable);
```

Comment:
Same comment as for the `CmmnCommandInvoker`

Reply:
Done